src/api/uploads/MultiputPart.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @flow
+ * @file Multiput upload part
+ * @author Box
+ */
+import Base from '../Base';
+
+const PART_STATE_NOT_STARTED: 0 = 0;
+const PART_STATE_COMPUTING_DIGEST: 1 = 1;
+const PART_STATE_DIGEST_READY: 2 = 2;
+const PART_STATE_UPLOADING: 3 = 3;
+const PART_STATE_UPLOADED: 4 = 4;
+
+class MultiputPart extends Base {
+    index: number;
+    numDigestRetriesPerformed: number;
+    numUploadRetriesPerformed: number;
+    offset: number;
+    sha256: ?string;
+    size: number;
+    state:
+        | typeof PART_STATE_NOT_STARTED
+        | typeof PART_STATE_COMPUTING_DIGEST
+        | typeof PART_STATE_DIGEST_READY
+        | typeof PART_STATE_UPLOADING
+        | typeof PART_STATE_UPLOADED;
+    timing: Object;
+    uploadedBytes: number;
+
+    /**
+     * [constructor]
+     *
+     * @param {object} options
```

Comment:
To stay consistent, we use {Object} and {Array} for non-primitives and {string} {number} for primitives

Reply:
done